pages/api/landingPage.js

```diff
@@ -36,17 +36,19 @@ async function updateContent(req, res) {
   }
 }
 
-
 async function readContent(req, res) {
   var userData;
-  try {
-    if (req.query.id != null || req.query.id != '' ) {
+  try { 
+    if (req.query.id  && (req.query.id != null || req.query.id != '' )) {
```

Comment:
Curious about this line.

The first part of the check should make sure `req.query.id` is not null and not undefined. Not sure why you need to check again for not null

Reply:
Typo